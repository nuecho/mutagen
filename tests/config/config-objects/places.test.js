cfgObjectTests("places", "places-config.json", {checkMandatoryProperties: false});