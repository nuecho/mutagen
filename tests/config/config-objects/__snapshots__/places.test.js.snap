// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen places config import-export] should create the places: the places have been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ folder => type: 'folder', owner: 'tenant/Environment', path: 'placeSite'
+ physicalswitch => placePhysicalSwitch
+ script => Environment/placeCapacityRule
+ switch => Environment/placeSwitch
+ dn => number: 'placeDN', switch: 'placeSwitch', type: 'cellular', tenant: 'Environment'
+ place => Environment/cfgObjectTest
Completed. 6 object(s) imported.
",
}
`;

exports[`[mutagen places config import-export] should update the modified places: the modified places have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ place => Environment/cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen places config import-export] should update the same places: the places have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ folder => type: 'folder', owner: 'tenant/Environment', path: 'placeSite'
~ physicalswitch => placePhysicalSwitch
~ script => Environment/placeCapacityRule
~ switch => Environment/placeSwitch
~ dn => number: 'placeDN', switch: 'placeSwitch', type: 'cellular', tenant: 'Environment'
~ place => Environment/cfgObjectTest
Completed. 6 object(s) imported.
",
}
`;
